### PR TITLE
Add invitation

### PR DESCRIPTION
**Pull request by entropitor**
I've added an invitation to our first meeting to the website.

### CODE REVIEW

File: 0meeting-en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Why open data?
+The current generation of students was raised with the internet. Their smartphone is always on. And they want to be able to see and do everything on this small device. It's impossible for educational institutions to provide and maintain all tools that are useful for students. But the data that is necessary to build them is property of the institution and often not publicly accessible. Therefore, an entrepreneurial student or staff member can not just build it themselves.
+
+That's why we believe we need "open data". Open data is data that is freely accessible and can be used by everyone. It can stimulate innovative applications in the educational sector. It also offers the opportunity for enterpreneurial students and/or staff members to help build towards a better college or university.
+
+We hope to see you February the 15th. If you have questions about this initiative or if you can't make it, but you want to be kept informed, just let us know!
+
+Jens Claes  
+President Student Innovations vzw (makers of Quivr)
```
Comment (jhoobergs) [inline 0meeting-en.md]: `makers` -> `creators` ?

File: 0meeting-en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Join us on **February 15th, at 5 pm near Brussels central station**. On this first meeting, you can meet other people interested in open data in education in Belgium, as well as people who have experience with open data in general. Together, we would like to figure out which data can be interesting to open up. We would also like to get an overview of all things that are open already.
+
+If you're interested, register at [our eventbrite](https://www.eventbrite.nl/e/open-education-kickoff-meeting-registration-29433969796). And be sure to share this with everyone who could be interested as well.
+
+### Why open data?
+The current generation of students was raised with the internet. Their smartphone is always on. And they want to be able to see and do everything on this small device. It's impossible for educational institutions to provide and maintain all tools that are useful for students. But the data that is necessary to build them is property of the institution and often not publicly accessible. Therefore, an entrepreneurial student or staff member can not just build it themselves.
```
Comment (jhoobergs) [inline 0meeting-en.md]: `Their smartphone is always on` -> `Their smartphone is always close` or something like that

**Review by jhoobergs — COMMENTED** [review]
Maybe we should still make it more clear that it isn't (in the first place) about opening up results from research ?

**Comment by hermansje** [comment]
@jhoobergs I changed some details
I think 'life on campus could be improved by a digital application' in the first sentence and the mention of 'tools' in the why section make clear what the first goal is.

**Pull request merged**